### ISSUE TITLE
15 min counts not uploading?

### ISSUE THREAD

**Issue by chrisvanswaay** [BiologicalRecordsCentre/ABLE#686]
@kazlauskis @DavidRoy 
During my trip through Spain the app keeps acting strange. The last week we did counts every day, I uploaded them in the evening, but the next day they would be back in the app in my list of counts to upload. However I could see them on the website, so the upload went fine. This stopped however the last two days and the app seemed to work fine, so uploaded counts did not come back in the list of counts to upload. However there are still 15 old counts in the 'waiting for upload', which have been uploaded (=they are visible on the website). What should I do with them? Delete them? Or upload them again?

Today is different. I uploaded my counts, and in the app they moved from 'waiting for upload' to 'uploaded', but I don't see them on the website after 2 hours. Is there something wrong? So far they would always appear within minutes.
v1.26.3 (350), android 14, pixel 8

**Comment by kazlauskis** [comment]
@chrisvanswaay You can safely reupload those pending records. This can happen, usually due to network or similar issues. The app is designed to detect duplicates and merge them. 

Regarding the uploaded but missing records, I had a quick look and couldn't see anything odd. There are no records I can see from yesterday. Did you use the 'Training' option? If you see those records uploaded, then you should be able to open them in the Upload tab and open the link to the website. Can you send me links to the website of those uploaded records?

<img width="421" alt="Screenshot 2024-06-04 at 18 10 18" src="https://github.com/BiologicalRecordsCentre/ABLE/assets/7383162/c4b0056b-5ee4-4f8f-87ed-fbc2155bea73">

**Comment by chrisvanswaay** [comment]
@kazlauskis Thanks, the counts keep coming back, but at least I know I can upload them safely again.
If I go to the website it still looks like my last count was on 2 June (today is 5 June):
![image](https://github.com/BiologicalRecordsCentre/ABLE/assets/25886005/0343dd53-e80b-44a6-a027-18fdd8db31fc)

But indeed using the link you pointed me to, I can see the counts of 3 June and even the ones I uploaded just this afternoon:
3 June: https://butterfly-monitoring.net/mydata/samples/edit?sample_id=25998923 (however the endtime is wrong, it was just 15 minutes and I finished the count, but uploaded in the evening - actually, the end time might be the upload time).

Today 5 June: https://butterfly-monitoring.net/mydata/samples/edit?sample_id=26031185
Here the endtime seems ok.

So apparently the counts are there, but the website only shows counts up to three days ago. Not sure who should deal with those. @JimBacon or @johnvanbreda ?

**Comment by kazlauskis** [comment]
@chrisvanswaay I can see there have been a number of surveys since 2 June, including the ones from 3 June, in your account now. Do you still experience this? This could be a glitch or a stale date shown in the website report.

**Comment by chrisvanswaay** [comment]
@kazlauskis I did quiet a few counts, but whatever I tried, I could not upload them. Or better: I could push the upload button, the count would disappear and appear to be uploaded, but a few seconds later it re-appeared as not uploaded. This was during the trip, so on wifi in hotels. We just got home and I gave it another try, and now everything worked without a problem, all counts uploaded fine. 

I think it is good to realise that if the internet connection is not perfect (as often with wifi in a hotel) then the upload gives problems. With a good private internet connection it all works fine.
